admin_guide/manage_scc.adoc

```diff
@@ -277,6 +282,10 @@ Then, add the service account to the `privileged` SCC.
 $ oadm policy add-scc-to-user privileged system:serviceaccount:myproject:mysvcacct
 ----
 
+The next step is to ensure that resource is being created in behalf of the
```

Comment:
Then, ensure that the resource is being created on behalf of the service account. To do so, set the . . .